composer.json

```diff
@@ -259,7 +259,7 @@
     "extra": {
         "symfony": {
             "allow-contrib": false,
-            "require": "5.4.*"
+            "require": "6.3.*"
```

Comment:
I had to change this, as there's no `symfony/clock` for Symfony `5.4`. I'll open a PR bumping Symfony's minimal version to `6.3` before we merge this PR.

Reply:
Resolved by merging #15327